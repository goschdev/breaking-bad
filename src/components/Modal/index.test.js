import React from "react"
import renderer from "react-test-renderer"

import { Modal } from "."

describe("[Component] Modal", () => {
  const props = {
    close: () => {},
  }
  it("Should renderer", () => {
    const tree = renderer.create(<Modal {...props}>Content</Modal>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
