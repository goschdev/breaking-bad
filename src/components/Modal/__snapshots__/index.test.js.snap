// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Modal Should renderer 1`] = `
<div
  aria-hidden={true}
  className="sc-AxjAm lejmDe"
>
  <button
    className="sc-AxirZ fGoBTZ"
    onClick={[Function]}
  >
    Close
  </button>
  <section
    className="sc-AxiKw iqcEQO"
  >
    <button
      className="sc-AxhCb qOIsM"
      onClick={[Function]}
      title="Close"
    />
    Content
  </section>
</div>
`;
